uber/templates/regform.html

```diff
@@ -480,14 +479,12 @@ <h4 class="h4-inline">
     <div class="col-sm-6">
         <input type="text" name="cellphone" id="cellphone" value="{{ attendee.cellphone }}" class="form-control" placeholder="Your Personal Cellphone Number">
     </div>
-    {% if not attendee.is_dealer %}
```

Comment:
You found the right place for this, but your instinct was right - we don't want to remove this in _all_ cases, only in cases where an admin is editing their information.

Probably the best way to solve this is to change this if-check to show these fields if someone has admin access (an easy way to do this is just check the page path) and to check if `no_cellphone` is true (because if an admin sets it, we don't want it removed whenever the attendee saves their info).